### PR TITLE
rust: add changelog doc to Afterburn and coreos-installer

### PR DESCRIPTION
**Pull request by bgilbert**
Update the release notes in a separate PR from the release itself, since `cargo release` creates a tag.  It'd be annoying to have to move the tag locally after addressing PR comments.

https://github.com/coreos/afterburn/pull/767
https://github.com/coreos/coreos-installer/pull/905

### CODE REVIEW

**Comment by jlebon** [comment]
Hmm, the double PRs is unfortunate overhead here. Would an alternative be to run `cargo release` with `--no-tag` and only create the tag after merging the release PR?

**Comment by bgilbert** [comment]
We could disable tagging in `Cargo.toml`, yeah.  I'd rather we not create the tag by hand, since a bad tag can't be fixed.  We could add a `tag_release.sh` script as in Ignition and Butane, though.  Is it worth it?  I don't see an additional PR as a big deal.

**Review by jlebon — APPROVED** [review]
> Is it worth it? I don't see an additional PR as a big deal.

When I say "overhead", I'm thinking of the wasted time and resources waiting for required CI to pass on the docs-only PR. We really should add file filters in all our CIs though to optimize that.

Anyway, LGTM as is too!

**Comment by bgilbert** [comment]
There's [some subtlety](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/troubleshooting-required-status-checks#handling-skipped-but-required-checks) to using path filters with required jobs, FWIW.  I'll go ahead and merge for now; we can always force-merge the release PR if time is a factor, and/or change the policy later if we want.

**Pull request merged**
